src/resources/language/de.json

```diff
@@ -174,7 +174,8 @@
           "UNIT": " Uhr"
         },
         "PRICE": "Fahrtkosten",
-        "NO_TRANS_AVAILABLE": "Keine (weiteren) Fahrten vorhanden"
+        "NO_TRANS_AVAILABLE": "Keine (weiteren) Fahrten vorhanden",
+        "REFUNDED": "erstattet"
```

Comment:
Please move this into the `-extensions.json` file. I separated my own from their translations to be able to update theirs easily, in case they change. The translation files are both loaded and deeply merged, all that changes is the capitalization of the root JSON object.